src/apps/btc/btc_sign.c

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Shift Cryptosecurity AG
+// Copyright 2020 Shift Cryptosecurity AG
```

Comment:
I'd keep the line as is and add a new with with 2020 Shift Crypto AG.

Reply:
oops of course.